hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -173,7 +176,11 @@ var (
 
 	etcdManualInterventionRequiredMetricDesc = prometheus.NewDesc(
 		EtcdManualInterventionRequiredMetricName, etcdManualInterventionRequiredMetricHelp,
-		append(hclusterLabels, "rosa_environment", "rosa_id"), nil)
+		append(hclusterLabels, "environment", "internal_id"), nil)
```

Comment:
how does this change relates to the PR?

Reply:
I see is for consistency below

Reply:
Would this break any consumption? cc @typeid

Reply:
This would not break anything on our side. Looks fine to me.